worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +43,40 @@ int main()
 
 	// TODO: implement the rest of the game
 
+	// Initialising variables
+	int guessesLeft = 4;
+	string guess;
+
+	// Game loop (incomplete)
+	while (guessesLeft > 0)
+	{
+		bool notGuessed = true;
+		while (notGuessed)
+		{
+			// Player input prompt
+			cout << "Password required";
+			cin >> guess;
+
+			// Checks if the guess is correct
+			if (guess == secret)
+			{
+				cout << "Access granted" << endl;
+				notGuessed = false;
+			}
+
+			// If the guess is wrong, reduce tries left by 1
```

Comment:
You're missing the check that the guessed word is in the list of options, and the calculation of the similarity score